response_operations_ui/controllers/survey_controllers.py

```diff
@@ -87,7 +87,7 @@ def get_survey_ci_classifier(survey_id):
 
 def get_surveys_list():
     logger.debug('Retrieving surveys list')
-    url = f'{app.config["SURVEY_URL"]}/surveys'
+    url = f'{app.config["SURVEY_URL"]}/surveys/surveytype/Business'
```

Comment:
:art: This capitalisation is odd

Reply:
I'd agree. Should probably be lowercased like the rest of the url

Reply:
It looks like the [survey service expects](https://github.com/ONSdigital/rm-survey-service/blob/c8963f9d3f49eacb565c8349e1b188d440849512/models/surveys.go#L518) a title case string. And the service would require some extra implementation to be case insensitive. 

As this isn't a public facing API we can probably keep this as is for now?